plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -45,7 +45,26 @@
     
     <library
           id="objectify"
-          group="appengine,flexible"
+          group="appengine"
+          name="Objectify"
+          tooltip="%objectify.tooltip"
+          dependencies="include"
+          siteUri="https://github.com/objectify/objectify/wiki" >
+      <libraryDependency id="appengine-api" />
+      <libraryFile
+          pinned="true"
```

Comment:
We could consider allowing wildcards or ranges in the version to make this kind of thing a bit more general (e.g., `[5.1,5.2)`).